clang/test/Driver/sparc-fix.c

```diff
@@ -0,0 +1,12 @@
+// RUN: %clang -target sparc -mfix-gr712rc -### %s 2> %t
```

Comment:
`--target=` for new tests. `-target ` has been deprecated since around Clang 3.4